cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -278,6 +278,64 @@ func (w *wizard) vote() {
 	fmt.Printf("\nVote success tx=0x%x\n", []byte(result))
 }
 
+func (w *wizard) voteOnProposal() {
+	if w.offchain {
+		glog.Error("Cannot vote in 'offchain' mode")
+		return
+	}
+
+	fmt.Print("Enter the proposal ID you want to vote on -")
+	proposalID := w.readStringAndValidate(func(in string) (string, error) {
+		if _, ok := new(big.Int).SetString(in, 10); !ok {
+			return "", fmt.Errorf("invalid proposal ID id=%v", in)
+		}
+		return in, nil
+	})
+
+	var (
+		confirm = "n"
+		choice  = types.ProposalVoteChoice(-1)
+	)
+
+	for confirm == "n" {
+		w.showProposalVoteChoices()
+
+		for {
+			fmt.Printf("Enter the ID of the choice you want to vote for -")
+			choice = types.ProposalVoteChoice(w.readInt())
+			if choice.IsValid() {
+				break
+			}
+			fmt.Println("Must enter a valid ID")
+		}
+
+		fmt.Printf("Are you sure you want to vote \"%v\"? (y/n) -", choice.String())
+		confirm = w.readStringYesOrNo()
+	}
+
+	fmt.Printf("Do you want to provide a reason for your vote? (y/n) -")
```

Comment:
Would be awesome to display these in the dashboard at some point!

Reply:
Mike will be adding it to his [vote tracker](https://tools.livepeer.cloud/vote/history). At the moment it is just me and Titan setting the reason, but with this PR we might see a lot more Orchestrators add a reason to their vote.